pyccl/ccl_mass_conversion.i

```diff
@@ -0,0 +1,33 @@
+%module ccl_mass_conversion
+
+%{
+/* put additional #includes here */
+%}
+
+%include "../include/ccl_mass_conversion.h"
+
+// Enable vectorised arguments for arrays
+%apply (double* IN_ARRAY1, int DIM1) {(double* c_in, int nc)};
+%apply (int DIM1, double* ARGOUT_ARRAY1) {(int nout, double* output)};
+
+%feature("pythonprepend") %{
+    if numpy.shape(c_in) != (nout,):
+        raise CCLError("Input shape for `c` must match `(nout,)`!")
+%}
+
+%inline %{
+
+  void get_new_concentration_vec(ccl_cosmology *cosmo,
```

Comment:
The name of this file doesn't match the corresponding one in the C code. This is a convention we have been sticking to that helps other navigate the code base.